source/precalculus/source/06-TR/04.ptx

```diff
@@ -12,6 +12,135 @@
     <subsection>
     <title>Activities</title>
 
+      <remark>
+        <p>
+          Recall from the previous section that we can find values of trigonometric functions of right triangles. In this section, we can take what we know about right triangles and use the same idea to find exact values of trigonometric functions of special angles.
+        </p>
+      </remark>
+      
+      <definition xml:id="def-special-right-triangles">
+        <statement>
+          <p>
+            There are two special right triangle relationships that will continually appear throughout the study of mathematics.
+            <ul>
+              <li>
+                <p>
+                  <m>45-45-90</m> triangle
+                </p>
+              </li>
+              <li>
+                <p>
+                  <m>30-60-90</m> triangle
+                </p>
+              </li>
+            </ul>
+          </p>
+        </statement>
+      </definition>
+
+      <activity xml:id="exploring-45-45-90-triangle">
+        <introduction>
+          <p>
+            In this activity, let's explore the relationship of the <m>45-45-90</m> special right triangle.
+        </introduction>
```

Comment:
```suggestion
        <introduction>
          <p>
            In this activity, let's explore the relationship of the <m>45-45-90</m> special right triangle.
          </p>
        </introduction>
```

If you look at the PreTeXt output, it says
```
critical: Failed to build.  Exiting...
critical: Opening and ending tag mismatch: p line 43 and introduction, line 45, column 24
```

Which is telling us that the `<p>` opened on line 43 is not being closed before the `</introduction>` tag on line 45.  So commit this suggestion and this error should be fixed.